src/text/masked.rs

```diff
@@ -45,15 +45,15 @@ impl<'a> Masked<'a> {
 
 impl fmt::Debug for Masked<'_> {
     /// Debug representation of a masked string is the underlying string
-    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        f.write_str(&self.inner)
+    fn fmt(&self, formatter: &mut fmt::Formatter<'_>) -> fmt::Result {
+        formatter.pad(&self.inner)
```

Comment:
Why implement Display / Formatter stuff ourselves when we can just use the existing Display implementation?

Reply:
because we know what the contents contain and can skip the extra layer of indirection. Happy to revert.

Reply:
But now we need to care how to implement `Display for str` well. The point of this PR is to not care about the implementation and use the Rust made implementation for the given thing we want to Display.

Reply:
402c5b45a